.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
   - export PATH="$HOME/miniconda/bin:$PATH"
   - hash -r
   - conda config --set always_yes yes --set changeps1 no
-  - conda config --add channels https://conda.anaconda.org/biocore
+  - conda config --add channels https://conda.anaconda.org/biocore https://conda.anaconda.org/mortonjt
   # Update conda itself
```

Comment:
this should be separated into two line (and add mine as well? probably need it in future ):
conda config --add channels https://conda.anaconda.org/biocore
conda config --add channels https://conda.anaconda.org/mortonjt
conda config --add channels https://conda.anaconda.org/zechxu